### DISCUSSION TITLE
Custom CSS

### DISCUSSION THREAD

**Discussion by mahmudmihon** [ant-design-blazor/ant-design-blazor#3482] [Q&A] [answered]
How can I apply custom CSS to components?

**Comment by ElderJames** [comment, marked as answer]
Hello @mahmudmihon , 

- If you are using css isolation, you need to add a div to warp the components, then use the css  selector like `div::ant-btn `.
- Otherwise, you can just use global css to change the style of components.

Here is the document: 
https://learn.microsoft.com/en-us/aspnet/core/blazor/components/css-isolation?view=aspnetcore-5.0
